### PR TITLE
marcnoon added restaraunts for lawndale 

### PR DESCRIPTION
**Pull request by marcnoon**
I added some restaurants for Lawndale.  I'm trying to merge into master.  I want to compare my feature branch with the master branch.  base is master and compare: is my branch.
- is anyone interested in good food and drinks in Los Angeles area?
  @githubteacher  suggested I added this, what do you think about that sir?

### CODE REVIEW

File: marcnoon-lawndale [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Lawndale, CA
```
Comment (githubteacher) [inline marcnoon-lawndale:1]: @marcnoon: I'll be in CA next week! What part is Lawndale in? How is the weather this time of year?

File: marcnoon-lawndale [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+### Restaurants
+- Marie Calendar
+- Golden Dragon
```
Comment (githubteacher) [inline marcnoon-lawndale:5]: @marcnoon: what kind of food is this?

File: marcnoon-lawndale (hunk 1 above)
Comment (marcnoon) [inline marcnoon-lawndale:1]: Its very hot ... 88 deg ☀️

File: marcnoon-lawndale (hunk 2 above)
Comment (marcnoon) [inline marcnoon-lawndale:5]: Goldren Dragon is a very good Chinese Restaurant in downtown LA 🐲 .

**Pull request merged**